File: Core/Models/PessoaFisica.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace vega.Core.Models
{
    public class PessoaFisica
    {
        public int Id { get; set; }
        [Required]
        [StringLength(50)]
        public string Nome { get; set; }
        [Required]
        [StringLength(50)]
        public string SobreNome { get; set; }
        [Required]
        public DateTime DataNascimento { get; set; }
        [Required]
        public string CPF { get; set; }
        [Required]
        public string Estado { get; set; }
        [Required]
        public string Cidade { get; set; }
         [StringLength(200)]
        public string Logradouro { get; set; }
    }
}

File: ClientApp/app/models/pessoaFisica.ts
import { Endereco } from './endereco';

export interface PessoaFisica {
    id: number;
    nome: string;
    sobreNome: string;
    cpf: string;
    endereco: Endereco;
  }

  export interface SavePessoaFisica {
    id: number;
    nome: string;
    sobreNome: string;
    cpf: string;
    endereco: Endereco;
  }
